upup/models/cloudup/_aws/topologies/_topology_private/bastion.yaml

```diff
@@ -102,7 +112,7 @@ launchConfiguration/bastion.{{ ClusterName }}:
    - securityGroup/bastion.{{ ClusterName }}
   iamInstanceProfile: iamInstanceProfile/masters.{{ ClusterName }}
   imageId: {{ GetBastionImageId }}
-  instanceType: t2.small
+  instanceType: {{ GetBastionMachineType }}
```

Comment:
Can we be certain this function will always return a value? I noticed on line 559 in `cluster.go` https://github.com/kubernetes/kops/pull/906/files#diff-843e01eb04cf09979da2bd7a655b3526R559 we are just returning the value that is initialized in the `cluster.Spec`

Are we positive there will always be something there?